Terminal.Gui/View/Adornment/Border.cs

```diff
@@ -375,7 +375,7 @@ private void Application_UnGrabbingMouse (object sender, GrabMouseEventArgs e)
         }
     }
 
-#endregion Mouse Support
+    #endregion Mouse Support
```

Comment:
Just an optional/discretional request for visibility:

Regions are exceptions to formatting rules, particularly around sorting, and `partial` types were _partially_ (ha) created as a "better" means of doing regions.

It may be worth considering splitting such things out into nested files using names like (for this one) `Border.Mouse.cs` style of naming, where files named X.Y.cs get nested under X.cs in the solution explorer (unless you turn off nesting like a crazy person).

You can even just put multiple type parts for the same type in the same file, if you want, which still achieves the same result as regions, but doesn't hide things from someone looking for stuff, as regions have an unfortunate tendency to do, being purely preprocessor elements.